backend/test/Test/API/Util.hs

```diff
@@ -0,0 +1,41 @@
+-- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: AGPL-3.0-or-later
+
+module Test.API.Util
+  ( app
+  , clientEnv
+  , apiTry
+  , errorWithStatus
+  ) where
+
+import Universum
+
+import Control.Exception (throwIO)
+import Network.HTTP.Client (defaultManagerSettings, newManager)
+import Network.HTTP.Types (Status)
+import Servant (Application, serve)
+import Servant.Client
+  (BaseUrl(baseUrlPort), ClientEnv, ClientError(..), ClientM, mkClientEnv, parseBaseUrl,
+  responseStatusCode, runClientM)
+
+import Edna.Setup (EdnaContext)
+import Edna.Web.API (ednaAPI)
+import Edna.Web.Server (ednaServer)
+
+app :: EdnaContext -> IO Application
+app ctx = return $ serve ednaAPI $ ednaServer ctx
+
+clientEnv :: Int -> IO ClientEnv
```

Comment:
[Here](https://hackage.haskell.org/package/http-client-0.7.8/docs/Network-HTTP-Client.html#v:newManager) they say:
> Creating a new `Manager` is a relatively expensive operation, you are advised to share a single `Manager` between requests instead.

And create it for each individual unit test, it sounds wrong. So I think we should:
1. Add a comment here saying that `clientEnv` is relatively expensive.
2. Update `LibrarySpec` to create it only once. Instead of `SpecWith EdnaContext` we can have `SpecWith (EdnaContext, ClientEnv)`.

Reply:
Apparently we can use `aroundAll` to solve this problem.